tests/integration/test_charm.py

```diff
@@ -23,8 +23,8 @@
 
 METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
 
-MYSQL_APP_NAME = "mysql"
-MYSQL_ROUTER_APP_NAME = "mysqlrouter"
+MYSQL_APP_NAME = "mysql-k8s"
```

Comment:
huge :facepalm: !

Reply:
No-no-no... the correct mysql-k8s charm used here, see the like 45, but the name was confusing here.